### PR TITLE
[quest] Added 14255 - Ethel Rethor (Horde)

### LINKED ISSUE #6159: Quest Issue for quest 14365 - Ethel Rethor (Desolace, Cataclysm Classic)
## Description
Alliance quest shows up on map minimap prematurely, requires a prerequisite quest to be completed first

## Wowhead link
Quest: https://www.wowhead.com/cata/quest=14365/ethel-rethor
Prerequisite: https://www.wowhead.com/cata/quest=14354/elunes-gifts

## Questie & WoW version
Questie 10.3.5 & Cataclysm Classic

### PR DESCRIPTION
**Pull request by TorpedoCXC**
Also added faction markers for this quest and the alliance quest of the same name.

## Issue references

Fixes #6159

### CODE REVIEW

File: Database/Corrections/cataQuestFixes.lua [hunk 1]
```diff
@@ -2009,6 +2009,11 @@ function CataQuestFixes.Load()
             [questKeys.preQuestSingle] = {14340},
             [questKeys.extraObjectives] = {{nil, Questie.ICON_TYPE_EVENT, l10n("Place the Arcane Charge in Balboa's path"), 0, {{"monster", 35759}}}},
         },
+        [14255] = { -- Ethel Rethor -- #6159
+            [questKeys.requiredRaces] = raceKeys.ALL_HORDE,
+            [questKeys.preQuestSingle] = {14189},
+            [questKeys.exclusiveTo] = {14256},
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:2015]:
```suggestion
            [questKeys.nextQuestInChain] = 14256,
```

**Review by BreakBB — COMMENTED** [review]
Looks fine to me and certainly is an improvement to the current state.

Thanks for taking the time to provide this PR 👍🏻

**Comment by TorpedoCXC** [comment]
@BreakBB You are welcome. I am still learning the details on how to improve the database though. Was my first PR for this project. :)

**Comment by BreakBB** [comment]
Feel free to hit us up on Discord if you have any questions @TorpedoCXC

**Comment by TorpedoCXC** [comment]
@BreakBB I tried joining the Discord server, but the link at the bottom of that page is no longer valid: https://github.com/Questie/Questie/wiki/Contributing

Please send me an updated invite link.

**Pull request merged**